README.md

```diff
@@ -54,6 +54,9 @@ and additional clean rules can be provided through the `clean-hook` variable.
 `make clean-images`  
 Best-effort to remove the last set of images that have been built using the scripts.
 
+`make shellcheck`  
+Check the shell syntax of all `*.sh` files tracked by the git.
```

Comment:
Just one small detail, otherwise LGTM
```suggestion
Check the shell syntax of all `*.sh` files tracked by the git repository.
```

Reply:
This was later changed, so no valid any more.